testgrid/cmd/updater/main.go

```diff
@@ -401,7 +514,7 @@ func ListBuilds(client *storage.Client, ctx context.Context, path string, builds
 		Prefix:    p,
 	})
 	fmt.Println("Looking in ", u, b, p)
-	var backwards []Build
+	var backwards Builds
```

Comment:
Is this sorted backwards? It looks normal to me.

Reply:
Done